bundles/org.openhab.ui.homebuilder/src/main/java/org/openhab/ui/homebuilder/internal/HomeBuilderDashboardTile.java

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) 2015-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.ui.homebuilder.internal;
+
+import org.openhab.ui.dashboard.DashboardTile;
+import org.osgi.service.http.HttpService;
+import org.osgi.service.http.NamespaceException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The dashboard tile and resource registering for HomeBuilder
+ *
+ * @author Kuba Wolanin - Initial contribution
+ *
+ */
+public class HomeBuilderDashboardTile implements DashboardTile {
+
+    @Override
+    public String getName() {
+        return "HomeBuilder";
+    }
+
+    @Override
+    public String getUrl() {
+        return "../homebuilder/index.html";
+    }
+
+    @Override
+    public String getOverlay() {
+        return null;
+    }
+
+    @Override
+    public String getImageUrl() {
+        return "../homebuilder/tile.png";
```

Comment:
I'd suggest to change the tile.png to be a screenshot of the tool as this is what we have for all other tiles as well. Introducing a new type of logo can always be tricky, so better refrain from it.

Reply:
Done!